apps/web/src/app/[locale]/(email)/waitlist/_components/waitlist-form.tsx

```diff
@@ -136,11 +136,11 @@ export function WaitlistForm() {
 
           <Button
             className="wl-form-button group relative mt-6 w-full overflow-hidden rounded-xl px-[2px] py-[2px] font-bold transition-shadow duration-300 hover:shadow-[0_0.5rem_2rem_-0.75rem_#3178c6] dark:hover:shadow-[0_0.5rem_2rem_-0.75rem_#5198f6]"
-            disabled={mutation.status === 'loading'}
+            disabled={mutation.status === 'pending'}
```

Comment:
https://tanstack.com/query/v5/docs/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading